monero-outreach-docs/translations/fr/archive/getting_started_fr.md

```diff
@@ -0,0 +1,197 @@
+# Un Starting Kit pour Aider Monero
+
+**“Une communauté bouillonnante, éclatée aux quatre coins de la planète, composée d’individus et de groupes unis par un projet décentralisé: Monero” - 21 août, 2019**
```

Comment:
La communauté de Monero est une communauté bouillonnante, éclatée aux quatre coins de la planète, composée d’individus et de groupes unis par un projet décentralisé

Reason: I feel that putting 'Monero' at the end move way to far the main topic of this sentence.

Reply:
Done!

Reply:
All good.